docs/developers/contributing.rst

```diff
@@ -61,8 +61,290 @@ Run tests
 
 Please refer to the document on :doc:`testing <testing>` to learn how to :ref:`run MicroOVN tests`.
 
+Contribute to our documentation
```

Comment:
I'd suggest not placing the docs contributing guide under docs/developers. Perhaps maintain the original contributing guide here as "Contribute to our code" and then a separate "Contribute to our documentation" under docs/.

(Or the approach we discussed earlier - HPC's approach)